runtime/src/server/middleware/index.ts

```diff
@@ -99,9 +99,14 @@ export function serve({ prefix, pathname, cache_control }: {
 	pathname?: string,
 	cache_control: string
 }) {
+	// Filter requests based on req.path.
 	const filter = pathname
 		? (req: Req) => req.path === pathname
-		: (req: Req) => req.path.startsWith(prefix);
+		/*
+			* [#1442](https://github.com/sveltejs/sapper/issues/1142)
```

Comment:
nit: it looks weird that this line is indented more than the previous line that starts the comment. also, usually I would just use the `//` syntax for a two line comment in the middle of the code

Reply:
Thank you for your feedback, I have withdrawn the pull request to improve it. :)

Reply:
you can update existing requests. you just make the change, run `git commit`, and then `git push` and it will update the pull request

Reply:
didnt' work out yet, will continue this evening..

Reply:
> you can update existing requests. you just make the change, run `git commit`, and then `git push` and it will update the pull request

I'll keep that in mind for any future pull requests I want to update. For now, I took a large detour on updating the pull request..